packages/core/src/constants/chainId.ts

```diff
@@ -30,13 +34,16 @@ export const MULTICALL_ADDRESSES = {
   [ChainId.Kovan]: '0x2cc8688c5f75e365aaeeb4ea8d6a480405a48d2a',
   [ChainId.BSC]: '0x41263cba59eb80dc200f3e2544eda4ed6a90e76c',
   [ChainId.xDai]: '0xb5b692a88bdfc81ca69dcb1d924f59f0413a602a',
+  [ChainId.Polygon]: '0xb5b692a88bdfc81ca69dcb1d924f59f0413a602a',
+  [ChainId.Mumbai]: '0xb5b692a88bdfc81ca69dcb1d924f59f0413a602a',
```

Comment:
I don't think those are proper addresses, but I can see proper once are listed here:
https://github.com/makerdao/multicall#multicall-contract-addresses

Reply:
I was sure I entered these, need to improve my copypasta skills!
Done now